plugin/db/mysql/mysql.go

```diff
@@ -162,6 +163,14 @@ func (driver *Driver) Execute(ctx context.Context, statement string) error {
 	}
 	defer tx.Rollback()
 
+	connID, err := driver.getConnID(ctx, tx)
+	if err != nil {
+		return err
+	}
+	// Save the thread ID of the latest executed transaction.
+	// It is used to filter the binlog events of the current transaction in the rollback SQL generation.
+	driver.lastTxnConnID = connID
```

Comment:
Any reason to store it in the driver?

Reply:
Otherwise, we must change the interface method `Execute`, which is MySQL only.